<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>性能测试工具</string>
  </property>
  <widget class="QPushButton" name="Memory">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>内存</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Mem_list">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>101</width>
     <height>631</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_run">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>最大值</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Max_mem">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
